packages/wrangler/src/api/startDevWorker/utils.ts

```diff
@@ -41,3 +44,158 @@ export function urlFromParts(
 
 	return url;
 }
+
+export function unwrapHook<T extends string | number | object>(
+	hook: Hook<T>
+): T | Promise<T>;
+export function unwrapHook<T extends string | number | object>(
+	hook: Hook<T> | undefined
+): T | Promise<T> | undefined;
+export function unwrapHook<T extends string | number | object>(hook: Hook<T>) {
+	return typeof hook === "function" ? hook() : hook;
+}
+
+export class MissingConfigError extends Error {
```

Comment:
would this be better placed in [errors.ts](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/errors.ts) ?